data/json/effects.json

```
@@ -2073,6 +2073,52 @@
     "rating": "bad",
     "base_mods": { "vomit_chance": [ 500 ] }
   },
+  {
+    "type": "effect_type",
+    "id": "bile_stink",
+    "name": [ "Bile Stink" ],
+    "desc": [ "You're splattered with a greasy layer of liquified putrescence." ],
+    "apply_message": "You're covered in a foul smelling fluid!",
+    "rating": "bad",
+    "resist_traits": [ "FLOWERS", "ROSEBUDS", "SAPROVORE", "EATDEAD" ],
```

What's the rationale for flowers and rosebuds to resist this?
```suggestion
    "resist_traits": [ "SAPROVORE", "EATDEAD" ],
```

---

The only thing the resistance affects is the vomit chance. The assumption being that one's natural scent would make it more tolerable. But I guess if you are used to one and not the other then it might not do anything, or it might just combine into something worse.